### PR TITLE
[as9716-32d] Modify fan/psu sys-led due to HW control led

### PR DESCRIPTION
**Pull request by jostar-yang**
Signed-off-by: Jostar Yang <jostar_yang@accton.com.tw>

FAN/PSU LED is controlled by HW now. So modify code to show auto mode.
SW will not access cpld for fan/psu led.

### CODE REVIEW

File: packages/platforms/accton/x86-64/as9716_32d/onlp/builds/x86_64_accton_as9716_32d/module/src/ledi.c [hunk 1]
```diff
@@ -126,18 +116,18 @@ static onlp_led_info_t linfo[] =
     },
     {
         { ONLP_LED_ID_CREATE(LED_FAN), "LED 3 (FAN LED)", 0 },
-        ONLP_LED_STATUS_PRESENT,
-        ONLP_LED_CAPS_ON_OFF |ONLP_LED_CAPS_GREEN| ONLP_LED_CAPS_ORANGE ,
+        ONLP_LED_STATUS_PRESENT,
```
Comment (paulmenzel) [inline packages/platforms/accton/x86-64/as9716_32d/onlp/builds/x86_64_accton_as9716_32d/module/src/ledi.c:119]: Please remove the trailing tab.

**Review by paulmenzel — COMMENTED** [review]
> FAN/PSU LED is controlled by HW now. So modify code to show auto mode.
> SW will not access cpld for fan/psu led.

I do not fully understand the first sentence. Was the board layout changed (schematics)?

**Comment by jostar-yang** [comment]
> > FAN/PSU LED is controlled by HW now. So modify code to show auto mode.
> > SW will not access cpld for fan/psu led.
> 
> I do not fully understand the first sentence. Was the board layout changed (schematics)?

Originally, FAN/PSU LED is design to let SW control. But after some customer check and discuss, we decide to let CPLD FW control. Board layout is not changed.

**Comment by paulmenzel** [comment]
> > > FAN/PSU LED is controlled by HW now. So modify code to show auto mode.
> > > SW will not access cpld for fan/psu led.
> > 
> > 
> > I do not fully understand the first sentence. Was the board layout changed (schematics)?
> 
> Originally, FAN/PSU LED is design to let SW control. But after some customer check and discuss, we decide to let CPLD FW control. Board layout is not changed.

Can you please amend the commit message, something like `git rebase -i origin/master`, and update the merge/pull request description too?

File: packages/platforms/accton/x86-64/as9716_32d/onlp/builds/x86_64_accton_as9716_32d/module/src/ledi.c (hunk 1 above)
Comment (jostar-yang) [reply packages/platforms/accton/x86-64/as9716_32d/onlp/builds/x86_64_accton_as9716_32d/module/src/ledi.c]: Done

**Comment by jostar-yang** [comment]
I has do "git commit --amend....".  Do you still need I do "git rebase -i ... "?

**Comment by paulmenzel** [comment]
> I has do "git commit --amend....". Do you still need I do "git rebase -i ... "?

(`git commit --amend` is basically a shortcut for `git rebase -i HEAD^`.)

What did you amend? I only see, that you added a second fix-up commit, which should have been squashed into the first one. Additionally, there is still no git commit message besides the summary and Signed-off-by line. Please add your answer to my question to the git commit message body.

**Comment by paulmenzel** [comment]
And please squash both commit: `git rebase -i HEAD~2`, and then change `pick` to `f` in the second line.

**Comment by jostar-yang** [comment]
I make a mistake on the commit when do git rebase. To avoid let thing become more complicated. I create new PR and close this one.

**Comment by paulmenzel** [comment]
Thank you for the update. It is now https://github.com/opencomputeproject/OpenNetworkLinux/pull/813.

Please try to learn git, so these problems do not happen in the future.

**Pull request closed without merging**